src/styles/blocks/star.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
it's a good practice to handle all possible cases:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/c2893d61-9cda-4c7e-bbe3-b46023eab066)
add styles for stars--1, stars--2, etc.
use `scss` loop to avoid duplication